### PR TITLE
Updating markdown files

### LINKED ISSUE #937: The way to launch Sugarizer locally with Chrome has changed
A recent discussion on sugarlabs mailing list has rightly brought this to our attention that the path{as mentioned in markdown files} via which we open sugarizer in our browser is actually incorrect.

Even I faced this issue where, upon running the following command:

`chrome --allow-file-access-from-files  index.html `

A blank page opens up, 

Screenshot (593)

to tackle this I always opened the index.html file navigating into the directory where it resides. This is however a heuristic approach and I presume not the right way to access sugarizer in browser locally.

### PR DESCRIPTION
**Pull request by ricknjacky**
fixes #937

### CODE REVIEW

**Comment by ricknjacky** [comment]
New Behavior: 

Screenshot (594)

**Comment by ricknjacky** [comment]
@llaske please have a look at this and share your thoughts.

**Comment by llaske** [comment]
@ricknjacky I wonder if there is no a more simple way in Chrome CLI to open a local file using relative path.
Did you try other way to do that? Did you explore CLI options in Chrome? It work in previous version, may be there is another way to do the same.

**Comment by ricknjacky** [comment]
> @ricknjacky I wonder if there is no a more simple way in Chrome CLI to open a local file using relative path.
> Did you try other way to do that? Did you explore CLI options in Chrome? It work in previous version, may be there is another way to do the same.

There is, we can use `http-server` 
We'd have to install this package accordingly.
`npm install http-server -g`

Now, in the sugarizer directory, to serve it on port 8000, we can type:

`http-server -p 8000`

Should we approach the problem by this method?

**Comment by llaske** [comment]
No it's worse, it need mean installing an HTTP server.
The idea is to be able to start Sugarizer development just by copying files in a local directory then using a browser and a text editor.

**Comment by ricknjacky** [comment]
> No it's worse, it mean installing an HTTP server.
> The idea is to be able to start Sugarizer development just by copying files in a local directory then using a browser and a text editor.

I think then, the way proposed above is the correct one.
I say this because, we don't know the directory user is cloning this repo into, hence mentioning the steps accordingly will be correct. What do you think? please share your suggestions.

File: README.md [hunk 1]
```diff
@@ -62,7 +62,16 @@ To run locally **Sugarizer Application into the Web Browser** (GNU Linux/Mac OS/
 
 For **Chrome**, close ALL running instances of Chrome and re-launch it using the command line:
 
- 	chrome --allow-file-access-from-files  index.html
+ 	chrome --allow-file-access-from-files file:\\\Directory:\sugarizer\index.html
```
Comment (llaske) [inline README.md:65]: Will be better to write `file:\\\<Directory>\sugarizer\index.html`

File: README.md [hunk 2]
```diff
@@ -62,7 +62,16 @@ To run locally **Sugarizer Application into the Web Browser** (GNU Linux/Mac OS/
...
+
+In windows, \ is the way path is written and / is the way for Linux/OSX backward and forward slashes differ here, hence path for Linux/MacOS will be:
+
+	chrome --allow-file-access-from-files file:///Directory:/sugarizer/index.html
+
+Directory in the path above is where you have stored/cloned your sugarizer repo into. Notice the path in the image below to get the gist of it.
```
Comment (llaske) [inline README.md:71]: Write `<Directory>` instead. This paragraph should be before the one mentioning difference between Windows and Linux/MacOS

File: README.md [hunk 3]
```diff
@@ -62,7 +62,16 @@ To run locally **Sugarizer Application into the Web Browser** (GNU Linux/Mac OS/
 
 For **Chrome**, close ALL running instances of Chrome and re-launch it using the command line:
 
- 	chrome --allow-file-access-from-files  index.html
+ 	chrome --allow-file-access-from-files file:\\\Directory:\sugarizer\index.html
+
+In windows, \ is the way path is written and / is the way for Linux/OSX backward and forward slashes differ here, hence path for Linux/MacOS will be:
```
Comment (llaske) [inline README.md:67]: Write Windows with an uppercase.
Write MacOS instead of OSX.

File: README.md [hunk 4]
```diff
@@ -62,7 +62,16 @@ To run locally **Sugarizer Application into the Web Browser** (GNU Linux/Mac OS/
...
+
+	chrome --allow-file-access-from-files file:///Directory:/sugarizer/index.html
```
Comment (llaske) [inline README.md:69]: Write `file:///<Directory>/sugarizer/index.html`.

File: README.md [hunk 5]
```diff
@@ -62,7 +62,16 @@ To run locally **Sugarizer Application into the Web Browser** (GNU Linux/Mac OS/
...
+
+Directory in the path above is where you have stored/cloned your sugarizer repo into. Notice the path in the image below to get the gist of it.
+
+![](images/Chrome-Sugarizer.png)
```
Comment (llaske) [inline README.md:73]: The image is useless.

File: docs/tutorial/VanillaJS/step0.md [hunk 6]
```diff
@@ -26,7 +26,9 @@ We will use **Chrome** in this tutorial but any other browser will work as well.
 
 To run Sugarizer locally in Chrome, close any running instances of Chrome and re-launch it using the command line:
 
-    chrome --allow-file-access-from-files index.html
+    chrome --allow-file-access-from-files file:\\\Directory:\sugarizer\index.html
```
Comment (llaske) [inline docs/tutorial/VanillaJS/step0.md:29]: See previous comments.

File: docs/tutorial/VueJS/step0.md [hunk 7]
```diff
@@ -25,7 +25,9 @@ We will use **Chrome** in this tutorial but any other browser will work as well.
 
 To run Sugarizer locally in Chrome, close any running instances of Chrome and re-launch it using the command line:
 
-    chrome --allow-file-access-from-files index.html
+    chrome --allow-file-access-from-files file:\\\Directory:\sugarizer\index.html
```
Comment (llaske) [inline docs/tutorial/VueJS/step0.md:28]: See previous comments.

**Review by llaske — CHANGES_REQUESTED** [review]
Okay guess that there is no CLI option in Chrome to avoid prefixing with `file:///<Directory>` ?
See my request changes below.

**Comment by ricknjacky** [comment]
> Okay guess that there is no CLI option in Chrome to avoid prefixing with `file:///<Directory>` ?

No, I searched for one like that, couldn't find one. Please do tell me if there is one in future 

> See my request changes below.

Committing them now. Thanks for the feedback.

**Comment by llaske** [comment]
Nice. Thanks.

**Pull request merged**